src/main/kotlin/io/titandata/titan/clients/Docker.kt

```diff
@@ -110,6 +110,11 @@ class Docker(private val executor: CommandExecutor) {
         return executor.exec(listOf("docker", "container", "rm", containerId))
     }
 
+    fun removeTitanImages(version: String): String {
+        val imageId = executor.exec(listOf("docker", "images", "titan:$version", "--format", "{{.ID}}")).trim()
+        return executor.exec(listOf("docker", "rmi", imageId, "-f"))
```

Comment:
I think we should be catching and ignoring errors here. If someone has by chance manually removed these images, I don't think we want uninstall to fail, right?

Reply:
We do this with the docker containers. It's been added for the image as well.